docs/USERS-GUIDE.md

```diff
@@ -1,40 +1,41 @@
-# DAPLink Users Guide
-There are three interfaces that DAPLink provides. These are drag-n-drop programming, a serial port and debugging support. In addition DAPLink firmware can be updated using the drag-n-drop programming interface of the bootloader.
+# DAPLink users guide
 
+DAPLink provides three interfaces. These are drag-and-drop programming, a serial port and debugging support. In addition, you can update DAPLink firmware using the drag-and-drop programming interface of the bootloader.
 
-## Drag-n-drop Programming
-Program the target microcontroller by copying or saving a file in one of the supported formats to the DAPLink drive. Upon completion the drive will re-mount. If a failure occurs then the file ``FAIL.TXT`` will appear on the drive containing information about the failure.
+## Drag-and-drop programming
+
+Program the target microcontroller by copying or saving a file in one of the supported formats to the DAPLink drive. Upon completion, the drive remounts. If a failure occurs, the file `FAIL.TXT` appears on the drive containing information about the failure.
 
 Supported file formats:
-* Raw binary file
-* Intel Hex
 
+- Raw binary file.
+- Intel Hex.
+
+## Serial port
 
-## Serial Port
 The serial port is connected directly to the target MCU allowing for bidirectional communication. It also allows the target to be reset by sending a break command over the serial port.
 
 Supported baud rates:
-* 9600
-* 14400
-* 19200
-* 28800
-* 38400
-* 56000
-* 57600
-* 115200
 
-Note - Most DAPLink implementations support other baud rates in addition to the ones listed here.
+- 9600.
+- 14400.
+- 19200.
+- 28800.
+- 38400.
+- 56000.
+- 57600.
+- 115200.
```

Comment:
Can you please remove the periods after the baud rates. That is a list of items, not sentences. It looks really weird with the period. Same for supported file formats above.

Reply:
That's fine, and makes sense, for sentences. But these are not sentences here. I've *never* seen technical documentation that uses periods in cases like this.